### PR TITLE
docs: added a link to Combobox on the Menu page 

### PR DESCRIPTION
**Pull request by SarthakD15**
#### Basic

- [x] Used plop (`npm run plop`) to create a new component.
- [x] PR has description.
- [x] New component is functional and uses Hooks.
- [x] Component is written in TypeScript.

#### Style

- [x] Styles are added to `NewComponent.modules.scss` file inside the `NewComponent` folder.
- [x] Component uses CSS Modules.
- [x] Design is compatible with [Monday Design System](https://design.monday.com/).
- [x] Styles are defined using [monday-ui-style](https://github.com/mondaycom/monday-ui-style) tokens
- [x] Displays correctly in all the themes

#### Storybook

- [x] Stories were added to `/src/components/NewComponent/__stories__/NewComponent.stories.mdx` file.
- [x] Stories include all flows of using the component.
- [x] Stories implemented using [vibe-storybook-components](https://github.com/mondaycom/vibe-storybook-components) components and tokens.

#### Tests

- [x] Tests are compliant with [TESTING_README.md](TESTING_README.md) instructions.

#### Accessibility

- [x] Component is accessible.
- [x] Component is compatible with [Vibe Accessibility Guidelines](https://style.monday.com/?path=/docs/foundations-accessibility--page)

### CODE REVIEW

File: src/components/Menu/Menu/__stories__/Menu.stories.mdx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import Menu from "../Menu";
+import { Link } from "vibe-storybook-components";
```
Comment (SergeyRoyt) [inline src/components/Menu/Menu/__stories__/Menu.stories.mdx:2]: Let's only remove this line, as Link is already imported on line 22 🙏

**Review by SergeyRoyt — CHANGES_REQUESTED** [review]
One small fix and ready to merge :)

**Review by SergeyRoyt — APPROVED** [review]
Looks good! Thanks and congrats with the first contribution!! 🤩

**Comment by SarthakD15** [comment]
Thank you @SergeyRoyt for the help !
Excited to contribute more to monday.com !  :))

**Pull request merged**